please fix linting issue
```
numpyro/infer/inspect.py:135:75: W291 Trailing whitespace
    |
133 |     in the posterior due to a so-called "collider" variable ``c``. ``a``
134 |     and ``b`` both affect `c``, and so they are conditionally dependent
[13](https://github.com/pyro-ppl/numpyro/actions/runs/13310600270/job/37171943767?pr=1975#step:5:14)5 |     given a value of ``c``. This is called "moralization" in the graphical 
    |                                                                           ^ W291
136 |     model literature.::
    |
    = help: Remove trailing whitespace
```

--- a/numpyro/infer/inspect.py
+++ b/numpyro/infer/inspect.py
@@ -132,7 +132,7 @@ def get_dependencies(
     conditionally independent in the prior but conditionally dependent
     in the posterior due to a so-called "collider" variable ``c``. ``a``
     and ``b`` both affect `c``, and so they are conditionally dependent
-    given a value of ``c``. This is called "moralization" in the graphical 
+    given a value of ``c``. This is called "moralization" in the graphical
     model literature.::
 
         def model_2():

